_includes/selector.html

```diff
@@ -617,15 +617,14 @@
                     } else {
                         var pkgs_html = this.rapids_meta_pkgs.map(pkg => "<code>" + pkg + "</code>").join(", ");
                     }
-                    notes = [...notes, "The CUDA " + this.active_cuda_ver +
-                        " <code>Standard</code> selection contains the following packages:<br>" + pkgs_html];
+                    notes = [...notes, "The <code>Standard</code> selection contains the following packages:<br>" + pkgs_html];
```

Comment:
This message doesn't have to state the CUDA version anymore because the standard list of packages no longer depends on the CUDA version (cuSignal was CUDA 11 only but has been dropped).